bitrise.yml

```diff
@@ -385,10 +385,13 @@ workflows:
             set -e
             # debug log
             set -x
-            curl --location --retry 5 --output l10n-screenshots-dd.zip
-            "$MOZ_DERIVED_DATA_PATH"
+            echo "curl to Download derived data"
+            curl --location --retry 5 --output l10n-screenshots-dd.zip "https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/CWFhvx6-QCWaPJoYMz9d0w/artifacts/public/build/target.zip"
```

Comment:
trying with the latest url where the target.zip was stored, looking at the [taskcluster logs](https://firefox-ci-tc.services.mozilla.com/tasks/ICsyZk3pRvCA7H_YZ3diAQ/runs/0/logs/public/logs/live.log)(L35).
Once we check this works, I will go back to use the Env Var.

Reply:
Now the script is working downloading the target.zip: https://app.bitrise.io/build/2027abf1-edc0-4ca4-bda4-569d3f6638d7#?tab=log